frontend/src/container/Publications.jsx

```diff
@@ -182,6 +188,35 @@ export class Publications extends React.Component {
     }
   }
 
+  handleChangeNewPublicationTitle = e => {
+    this.setState({ newPublicationTitle: e.target.value })
+  }
+
+  handleTogglePublicationTitlePopup = (publication, publicationAsFileList, commentReset) => {
```

Comment:
if commentReset can be not sent, set it as optionnal

```suggestion
  handleTogglePublicationTitlePopup = (publication, publicationAsFileList, commentReset = undefined) => {
```

Reply:
done